README.md

```diff
@@ -51,13 +51,24 @@ rather than IP numbers in a Hyper-V environment. On CentOS systems,
 this requires the bind-utils package (updatedns requires the 'dig'
 program). The bind-utils package isn't otherwise necessary.
 
+- Install minimum version of Ruby (the system must have 1.9 or higher, but 2.2.6 is recommended)
```

Comment:
The minimum version of system Ruby must be higher than 1.8, because of the compilation compatibility issue you found. It cannot be 2.3 or higher, because that contains the incompatibility with OpenSSL 0.9.8. It is also preferable that a version is chosen that is currently supported.
2.2.6 was chosen because it is the newest release that is lower than 2.3. However, any version higher than 1.8 should work.